frontend/public/style/_common.scss

```diff
@@ -149,6 +149,10 @@
   white-space: pre-wrap;
 }
 
+.co-pre-line-wrap {
```

Comment:
We should call this just `.co-pre-line`, although it might be better to just use `.co-pre-wrap` for the table